evap/staff/views.py

```diff
@@ -132,11 +132,11 @@ def semester_course_operation(request, semester_id):
         if operation == 'revertToNew':
             helper_semester_course_operation_revert(request, courses)
         elif operation == 'prepare' or operation == 'reenableEditorReview':
-            helper_semester_course_operation_prepare(request, courses)
+            helper_semester_course_operation_prepare(request, courses, request.POST.get('send_email'))
         elif operation == 'approve':
             helper_semester_course_operation_approve(request, courses)
         elif operation == 'publish':
-            helper_semester_course_operation_publish(request, courses)
+            helper_semester_course_operation_publish(request, courses, request.POST.get('send_email'))
```

Comment:
please put `request.POST.get('send_email') == 'on'` into a variable and then hand over only a boolean